docs/iris/src/whatsnew/1.10.rst

```diff
@@ -0,0 +1,170 @@
+* Deprecated :data:`iris.fileformats.grib.hindcast_workaround` and :class:`iris.fileformats.grib.GribWrapper`. The class :class:`iris.fileformats.grib.message.GribMessage` provides alternative means of working with GRIB message instances.
+* Deprecated the module :mod:`iris.fileformats.ff`.  Please use the replacement
+  facilities in module :mod:`iris.fileformats.um` :
+
+  * :func:`iris.fileformats.um.um_to_pp` replaces :class:`iris.fileformats.ff.FF2PP`.
+  * :func:`iris.fileformats.um.load_cubes` replaces :func:`iris.fileformats.ff.load_cubes`.
+  * :func:`iris.fileformats.um.load_cubes_32bit_ieee` replaces :func:`iris.fileformats.ff.load_cubes_32bit_ieee`.
+
+  All other public components are generally deprecated and will be removed in a future release.
+* The :func:`iris.fileformats.pp.as_pairs` and :func:`iris.fileformats.grib.as_pairs` are deprecated.  These are replaced with :func:`iris.fileformats.pp.save_pairs_from_cube` and :func:`iris.fileformats.grib.save_pairs_from_cube`.
+* ``iris.fileformats.pp_packing`` has been hidden behind a deprecation interface. Please install `mo_pack <https://github.com/SciTools/mo_pack>`_ to use instead.
+* Deprecated logging functions (currently used only for rules logging):  
+ :data:`iris.config.iris.config.RULE_LOG_DIR`,
+ :data:`iris.config.iris.config.RULE_LOG_IGNORE` and
+ :data:`iris.fileformats.rules.log`.
+* Deprecated all the remaining text rules mechanisms:  
+ :class:`iris.fileformats.rules.DebugString`,
+ :class:`iris.fileformats.rules.CMAttribute`,
+ :class:`iris.fileformats.rules.CMCustomAttribute`,
+ :class:`iris.fileformats.rules.CoordAndDims`,
+ :class:`iris.fileformats.rules.Rule`,
+ :class:`iris.fileformats.rules.FunctionRule`,
+ :class:`iris.fileformats.rules.ProcedureRule`,
+ :class:`iris.fileformats.rules.RulesContainer` and
+ :func:`iris.fileformats.rules.calculate_forecast_period`.
+* Deprecated the custom pp save rules mechanism implemented by the functions :func:`iris.fileformats.pp.add_save_rules` and :func:`iris.fileformats.pp.reset_save_rules`. The functions :func:`iris.fileformats.pp.as_fields`, :func:`iris.fileformats.pp.as_pairs` and :func:`iris.fileformats.pp.save_fields` provide alternative means of achieving the same ends.
+
+Documentation Changes
+=====================
+* It is now clear that repeated values will form a group under :meth:`iris.cube.Cube.aggregated_by` even if they aren't consecutive. Hence, the documentation for :mod:`iris.cube` has been changed to reflect this.
+* The documentation for :meth:`iris.analysis.calculus.curl` has been updated for clarity.
+* False claims about :meth:`iris.fileformats.pp.save`, :meth:`iris.fileformats.pp.as_pairs`, and :meth:`iris.fileformats.pp.as_fields` being able to take instances of :class:`iris.cube.CubeList` as inputs have been removed.
+* A new code example, demonstrating the use of a quiver plot to display wind speeds over Lake Victoria, has been added to the gallery.
```

Comment:
Could usefully include a link : (to docs/iris/build/html/examples/Meteorology/wind_speed.html)